File: benchmarks/fixtures/polyglot-services/src/user_service.jl
struct User
    name::String
    email::String
end

function display(user::User)
    return "$(user.name) <$(user.email)>"
end

function create(params::Dict)
    return User(params[:name], params[:email])
end

user = create(Dict(:name => "Henry", :email => "henry@example.com"))
println(display(user))

File: benchmarks/fixtures/polyglot-services/src/user_service.R
display <- function(name, email) {
    paste0(name, " <", email, ">")
}

create <- function(params) {
    list(name = params$name, email = params$email)
}

user <- create(list(name = "Grace", email = "grace@example.com"))
cat(display(user$name, user$email), "\n")